app/build.gradle.kts

```diff
@@ -96,7 +96,10 @@ dependencies {
     implementation(project(":feature_data_edit"))
     implementation(project(":feature_records_filter"))
     implementation(project(":feature_goals"))
+    implementation(project(":wearrpc"))
 
+    implementation("com.google.android.gms:play-services-wearable:18.0.0")
```

Comment:
I hadn't thought of that.

Since WearOS **requires** Google Play Services to communicate with the phone, I presume an F-Droid build would be impossible. While Reddit's not the most reliable source, [this thread](https://www.reddit.com/r/WearOS/comments/16269vu/fdroid_for_wearos_other_apps_not_in_the_play_store/) seems to echo that sentiment.

Perhaps my second bullet point is the appropriate solution for both this issue and the concerning minSDK bump:
> Create multiple builds of Simple Time Tracker? One without WearOS for Android 5.0-7.0, and another with WearOS for >=8.0?

I believe the following could work:
1.  Pull all the Wear code currently in the `app` package into a separate Android library and have the manifests merge at build time.
2. I believe `build.gradle` can specify multiple production build targets similar to how it specifies debug vs. release build targets. If that's true, it should be possible to do one build with the wearos modules and another without.
3. The build without the wear modules could retain support for API 21 + remain valid for F-Droid.

I'm confident I can complete step 1 on my own. Steps 2-3, however, would need your help to make sure they work with your deployment strategy.

Reply:
> Create multiple builds of Simple Time Tracker? One without WearOS for Android 5.0-7.0, and another with WearOS for >=8.0?

Yes, this should work.

Reply:
Which steps towards creating separate builds do you want to complete before merging this PR vs. after?

Reply:
You can do step 1 before merge and I will do steps 2-3 after the merge. Or we can merge as is, and I will do steps 1-2-3. Whichever you'd prefer.